quickstarts/codestream/codestream/config.yml

```diff
@@ -40,4 +40,3 @@ documentation:
 website: https://codestream.com
 # Keywords for filtering / searching criteria in the UI
 keywords:
```

Comment:
If there aren't any keywords I think you may have to delete the field all together.

Reply:
maybe we can add a tag for "IDE"?

Reply:
@seemantk what do you think of this suggestion? Seems fair to me.